test/library/characterization/test_qubit_spectroscopy.py

```diff
@@ -180,11 +180,14 @@ def test_expdata_serialization(self):
         frequencies = np.linspace(freq01 - 10.0e6, freq01 + 10.0e6, 21)
         exp = QubitSpectroscopy([qubit], frequencies)
 
+        # Checking serialization of the experiment obj
+        self.assertRoundTripSerializable(exp)
```

Comment:
Why do we also need the check before `exp.run`?

Reply:
I wanted to do it for the completeness of the test as in the function `test_kerneled_expdata_serialization`. I will delete it as this is not what this PR meant to fix.